t/t3903-stash.sh

```diff
@@ -632,6 +632,47 @@ test_expect_success 'stash create - no changes' '
 	test_must_be_empty actual
 '
 
+test_expect_success 'stash create with --include-untracked' '
+    git init repo &&
+    cd repo &&
+    echo committed >file1 &&
+    git add file1 &&
+    git commit -m "initial commit" &&
+    echo staged >file2 &&
+    git add file2 &&
+    echo unstaged >file3 &&
+    echo untracked >untracked_file &&
+    STASH_ID=$(git stash create --include-untracked "test message") &&
+    git cat-file -p $STASH_ID >stash_commit &&
+    grep "test message" stash_commit &&
+    grep "parent" stash_commit | wc -l >parent_count &&
+    echo 3 >expect &&
+    test_cmp expect parent_count &&
```

Comment:
@SBhojani As pointed out by the test failures, you will want to use `grep parent stash_commit >parents && test_line_count = 3 parents` instead (see the documentation of [`test_line_count`](https://github.com/git/git/blob/v2.49.0/t/test-lib-functions.sh#L1039-L1060); `wc -l`'s output is not strictly portable).